### DISCUSSION TITLE
Locations (Countries) updates for existing installations

### DISCUSSION THREAD

**Discussion by eurobank** [mindstellar/shopclass#380] [Q&A]
I would like to ask how an already established and working site can update the country/cities etc etc DB, without deleting and reinstalling a location (country).

**Comment by navjottomer** [comment]
You can manage them in location settings, there are options to delete/edit them there.

**Comment by eurobank** [comment]
I do not see any settings in Locations to update anything. How can i merge the new ones and delete the old ones? 

I hope you do not mean one-by-one by hand?

**Reply by navjottomer** [reply]
You can manually edit them or delete them from the admin locations page, it's not possible to merge them again from the new list, it will cause errors.

**Reply by navjottomer** [reply]
> I do not see any settings in Locations to update anything. How can i merge the new ones and delete the old ones?
> 
> I hope you do not mean one-by-one by hand?

That's what I meant, for doing anything else You'll need a custom SQL query.

**Comment by eurobank** [comment]
I do not understand how i can find NEW (from the github repo) or how to find the ones that should be deleted.

Anyways, i can't even do that in a multi country site. 

So if i understand correctly, the ONLY way (easy way) is to Remove a country and install it again. But even doing that, will not have any meaning for future updates of locations, because you can't delete and install again and again and again a country.

**Comment by navjottomer** [comment]
Currently, locations are tied to listings, that's why it is difficult to apply an update on locations. Maybe in future, we only tie coordinate with listings which can give us some room.
